models/networking.k8s.io.ingress.js

```diff
@@ -109,16 +110,63 @@ export default {
 
   createDefaultService() {
     return (workloads) => {
-      const name = this.spec.backend?.serviceName;
+      const backend = get(this.spec, this.defaultBackendPath);
+      const serviceName = get(backend, this.serviceNamePath);
 
-      if ( !name ) {
+      if ( !serviceName ) {
         return null;
       }
 
       return {
-        name,
-        targetTo: this.targetTo(workloads, name)
+        name:     serviceName,
+        targetTo: this.targetTo(workloads, serviceName)
       };
     };
+  },
+
+  showPathType() {
+    const ingressExpandedSchema = this.$rootGetters['cluster/expandedSchema'](this.type);
+    const spec = ingressExpandedSchema?.expandedResourceFields?.spec;
+    const rules = spec?.expandedResourceFields?.rules;
+    const http = rules?.expandedSubType?.expandedResourceFields?.http;
+    const paths = http?.expandedResourceFields?.paths;
+    const pathType = paths?.expandedSubType?.expandedResourceFields?.pathType;
+
+    return !!pathType;
+  },
+
+  useNestedBackendField() {
+    const ingressExpandedSchema = this.$rootGetters['cluster/expandedSchema'](this.type);
+    const spec = ingressExpandedSchema?.expandedResourceFields?.spec;
+    const rules = spec?.expandedResourceFields?.rules;
+    const http = rules?.expandedSubType?.expandedResourceFields?.http;
+    const paths = http?.expandedResourceFields?.paths;
+    const backend = paths?.expandedSubType?.expandedResourceFields?.backend;
+    const service = backend?.expandedResourceFields?.service;
+    const name = service?.expandedResourceFields?.name;
+
+    return !!name;
+  },
```

Comment:
Fine for now but would be nice if this was less boilerplate in the model and more like a getter, e.g.  `getters['cluster/schemaHasField'](this.type, spec.rules[].http.paths[].backend.service.name)`